NDcPP_v2_2e.adoc

```diff
@@ -3603,24 +3658,42 @@ FIA_X509_EXT.1 X.509 Certificate Validation
 FIA_X509_EXT.2 X.509 Certificate Authentication
 ____
 
-*FCS_DTLSS_EXT.1.1* The TSF shall implement [selection: _DTLS 1.2 (RFC 6347), DTLS 1.0 (RFC 4347)_] supporting the following ciphersuites:
+*FCS_DTLSS_EXT.1.1* The TSF shall implement [selection: _DTLS 1.3 (RFC tbd), DTLS 1.2 (RFC 6347)_] and [selection: _DTLS 1.0 (RFC 4347), no additional DTLS version_] supporting the following ciphersuites:
```

Comment:
@mclearn Same as previous comment...
Per the MS Word doc v13d, the closing bracket should be after 'DTLS 1.0 (RFC 4347)'.

Reply:
See response above.

Reply:
agreed